doc/releases/changelog-dev.md

```diff
@@ -74,9 +74,10 @@
 * `qml.ctrl(qml.PauliX)` returns a `CNOT`, `Toffoli` or `MultiControlledX` instead of a `Controlled(PauliX)`.
   [(#4339)](https://github.com/PennyLaneAI/pennylane/pull/4339)
 
-* The experimental device interface is integrated with the `QNode` for jax-jit and torch.
+* The experimental device interface is integrated with the `QNode` for jax-jit, tensorflow and torch.
   [(#4352)](https://github.com/PennyLaneAI/pennylane/pull/4352)
   [(#4392)](https://github.com/PennyLaneAI/pennylane/pull/4392)
+  [(#4393)](https://github.com/PennyLaneAI/pennylane/pull/4393)
```

Comment:
We can also consider combining this entry with the one above (line 63).

Reply:
ah I forgot about that one! I'll put them together